packages/remote-react-renderer/src/RemoteRendererClient.tsx

```diff
@@ -9,26 +10,59 @@ import {
 import { connectRemoteIframeRef } from "@mittwald/flow-remote-core";
 import {
   type ComponentType,
+  type CSSProperties,
   type FC,
   useLayoutEffect,
+  useMemo,
   useRef,
   useState,
 } from "react";
-import { useMemo } from "react";
 import { reduce } from "remeda";
 
 export interface RemoteRendererProps {
   integrations?: RemoteComponentsMap<never>[];
   src: string;
+  renderErrorTimeout?: number;
```

Comment:
Passt nicht ganz, weil ein Error in dem Sinne ja hier nicht gerendert wird, sondern nur geworfen.

`loadingTimeout` oder nur `timeout`?